src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -54,6 +54,10 @@
             ChatColor.DARK_BLUE,
             ChatColor.DARK_PURPLE,
             ChatColor.LIGHT_PURPLE);
+    public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
+            {
```

Comment:
Why do you need a String[]

Reply:
i moved it from the tag command to FUtil. ask whoever made it

Reply:
h, just remove the String[] and its { and }